class-mapping.php

```diff
@@ -102,6 +102,38 @@ public function set_active( $active ) {
 	}
 
 	/**
+	 * Make this mapping the primary domain, eg. set the homeurl for the site
+	 *
+	 * @return bool|\WP_Error True if we created the old mapping, false if we didn't need to, or WP_Error if an error occurred
+	 */
+	public function make_primary() {
```

Comment:
Wondering if this should maybe a function instead of a method, given that it destroys the object.